tests/layer_tests/tensorflow_tests/test_tf_Loop.py

```diff
@@ -0,0 +1,105 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
```

Comment:
also please use TF2 High level API (Keras concept aka) for programming these models